standalone/matRad_compileStandalone.m

```diff
@@ -0,0 +1,55 @@
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Script to compile an executable matRad version with the matlab
+% application compiler.
+%
+% There will be no splash screen included in the executable if following 
+% the instructions below.
+% In order to compile an executable with splash screen call deploytool and
+% add the required files manually.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2015, Mark Bangert, on behalf of the matRad development team
+%
+% m.bangert@dkfz.de
+%
+% This file is part of matRad.
+%
+% matrad is free software: you can redistribute it and/or modify it under 
+% the terms of the GNU General Public License as published by the Free 
+% Software Foundation, either version 3 of the License, or (at your option)
+% any later version.
+%
+% matRad is distributed in the hope that it will be useful, but WITHOUT ANY
+% WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+% FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+% details.
+%
+% You should have received a copy of the GNU General Public License in the
+% file license.txt along with matRad. If not, see
+% <http://www.gnu.org/licenses/>.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+%% start the compiling process from the matRad root directory
+dirMatRad = pwd;                % set the matRad root directory
+dirOut = [pwd '\standalone\build\']; % specify the output folder
+
+if exist(dirOut)~= 7
+   mkdir(dirOut);
+end
+
+addpath('dicomImport');         % include the DICOM Import files in the dependency analysis
+
+
+mcc('-C','-o','matRad','-W','WinMain:matRad','-T','link:exe','-d',dirOut,...      
+    '-R','-logfile,consoleOutput.log','-v',[dirMatRad '\matRadGUI.m'],...
+    '-a',[dirMatRad '\dicomImport\DKFZ_Logo.png'],'-a',[dirMatRad '\dicomImport\matrad_logo.png'],...
+    '-a',[dirMatRad '\dicomImport\hlutDefault.mat'])
```

Comment:
Including the machine files in the compilation process did not work for me as I tried it. I think this is due to a "problem" with the dir command not searching additionally among the files added during the compilation process, but only in the directory the executable is in. To avoid this I either have to change the code (e.g. include an exception for a deployed application) or search for different solutions (I did not find one during a quick research i did right now).

Concerning the patient/phantom data: Sure we could do that. I only thought of it to be a good idea, but if you think different I will change that! I think it would be nice to have a seperate location where you could get the installer from instead of "hiding" it in a subdirectory of the development branch. There we could also provide the patient data, just to make it easier for people to access them.

What's your thoughts about that?